lib/elixir/lib/kernel/cli.ex

```diff
@@ -480,7 +480,8 @@ defmodule Kernel.CLI do
   defp filter_patterns(pattern) do
     pattern
     |> Path.wildcard()
-    |> :lists.usort()
+    |> Enum.sort()
+    |> Enum.uniq()
```

Comment:
Same here: I tried to look for a way to do this in only one pass, but I wasn't able to find it, which leads me to think if we need to add a function to the API. If there is interest, I can think of something and write a proposal to extend Enum.sort